feedback-module/src/lib/utils/googleAnalytics.js

```diff
@@ -0,0 +1,21 @@
+export function trackFutureResearch(ga) {
```

Comment:
Instead of passing in ga to all of these methods, could we refactor this into a hook and just have useGA4React there since they all need ga?

Reply:
done